doc/Cellpose-StarDist-Segmentation.ipynb

```diff
@@ -0,0 +1,454 @@
R2Z4tRq8UgnOKg5ii51+CtqJpXmgXPWEVJJHCZ3A6Auxfr58z/IMgMjlcspEz+hySQjl4i6JHxc0GRXP8iRrMv8og6To5ylJG1VTmhRl/VxgG43GvIhkKrDValWVIVkF0BEtL7dMlH800cqgJpJlniOVYT/yF7QDjolc2tTtsKTRpP4FlQkLG7HzI6PA7D0xMjKCk08+GalUCocOHcKePXtQLpfnkVLZP1u9vKekAu7XftCDgCxHslYul5XPM+9F3u+8j2WEPNCZy1cGL0l3Gal48h5jXVRdgbmAOL2/8p4yPWjIzBHSRG/qN/seqaO9hSKjvQpgCoOn2mxPTQeSPCceB8ZHl6ljEXoJN5vFE+88A4ULjgIAHnvGyTjlr6pdqePLiXapglgms7zfhQXAK5dn8/tqOP7MMbSWKAHJyp6RAJhMmkHKj36uzZxLxXBwcBDZbFaltgHQsaABc0qLzNXJ1ySpXKBpJufnMtqdZnAGkXABlos1ybe+iLJstVpVr0kYZC5U9l+qQXIsVKIqlYrql8kUKqOS9a0a5RwHqZ562h95XVinJMlBkOfIa7AQmNREE3T3AT+lkvUmEgls3LgRz3zmM5HP5/Hwww9jcnIStVqtoz69HVv9fKBh9gNTCjR9XLbP9Pd8KCmXy+o+5ndAzxEq1XM9OEgmvJe7OJmIPFV8/aGL4+e9w7ImEi/nWt9SlPcHXQbk98l230ZYIHodTd8N4vFQKqkTj/clmCnC4uHV61j3QAv1C2bfDzxjAk4mA6xQQuo16vCardmHoBUMr1Se54PtXHgOKmNL526wuh0ZNOgLiU4UJExmTqkyJRIJDAwMKFO4yYQo90GXeTzlQinTHFGN0ffqBuaUISpQMqqYi6VcpHWzPgBlMq1WqyiVSiq4g32UwVUMLMpkMsqc2W63Ua1WUalUOpQjmVJHJvnnfDFIi3MoSUWQUkflWM6DLEdzK9P0SEKiX0NJNuS11BFEGE33TxAZ1R9+bA868i+TyXTkrpWZGmznmtrlH5O9yyj2oL7ZvjP6e6qb5XK5Q2GXc236bjDZPbfGBebcRKSFgMeZ9YEuAJJs8t7g/cDvidxNTB+3TNPG41Re+T2Q9fO766fiR1gA+rj4OoVCcH7SlXK9zzuj3z1YdXCzWRx5VbXj2C//amNPctIuFVpHj876kq4yNAZTaC3hvhorm6IHgAsHVRweA7pLb6OT0ng8jqGhIeUzCswRJx0MHpILdyKRUKoS65TRu7oZXFcAGYUsA3+kudJxHOVPRzKi54tkIJKMNpZBITICX/r8SfLM8XCOqapytyYAKsKaZbpVlqSyJc24cr9yXXGW7ehqKomJiQzLa85z5XuJoLGY1GDZP1tb8vXk5CQef/xx5PN5HD161KoA+z1QyTIyq4NpvHp9ugJs6rPubkFS6jgOcrlch8Ip7zNeA3mvSD9l3mdsg2p+s9lUpJDn8D+/MyTfhNxAQnfvkC4G0togH7Lkfce22L8Ii4QbQ2xsXZ/74MCJxeDZdnFyHLTOO215+2TBcibkXytozcxg66fimJ7bERen/Y8a2it5D3nPg9dswnkqgHjFoU9WolVNSP1M7mEgF2Op8jCanlHFunKlJ7yXCxn937jIynycUnWRORblnvLArALEBZaKpCRpehopGUyiL6qsX6aTkuqhrt7yXJ1AybGzL1zwqSCbCJ7NNUI3zbNdknNpwpekWl4vWQdfk0CHMfH7feFspMz0uT4mCdO9yOtSLBaxe/durF+/HrFYDLlcDlNTU/PSJvndz7Kc3I7WT6W1uQSE6Xej0VCme5LDWCymSKg8Vwa1UZGX/s4ySl8SW6mk8uGJpn6CxFVuLsG2dUuHro4Cna4Cum/3Qh+uInTCicfhim1N+wl3ZAitw0e6SwMVYVXAzefxxGtikB7Ah/+8idFXxHq3lewSoHXoMOJbt/S7G0Z40zPwhP/+cmFVE1LCRhgkWZGwmURd10U2m0WhUFALmYzKNflcAnOLnExRA8yPKJZmfPaRpJOR6qwnnU53mPylAstz+Zk0y1MlZBmTKir991iGfaL5XSpIem5VAMrEXK1WlX8eiYOJTPmRUvle9431M5f71RGGdIWBrN/PDB+mHVlXu91GpVLB9PQ0xsbGMDIyglarhampqY6NCsKOXc6dJOhhHthMVgJd9ZTjJynldedDDQmkrtDqfqK833gO6+b3jQ9RfBiUhFLP98u54n+mLCOplUF78nw5L0z7xHKc+yjKfuFwEkm4g4WlT+/UBdxMBu1yud/d8IXnOIiPr0fr0OF+d2XVoD0zg7M/9CR++V+3YfishW2I0C94xRKcfK7f3ehEswnvqc1KlhurmpCaFClCJxEm86SujjKnJ83QejJ6Lpg0AXJR1c2EcqcjaYanWZLH6bMpt0FkPTJHqU5iSQC4+NfrdZUnUpbhQi2j+OlTxzrlfEgzaCKR6Ej5pBMv1i/Nnn7XwE9llO/9HiT8zpXHFktG9XvHVg/9Z6kQkhDq95ok9jKCu16vo1gsYnp6GgMDA2pTg14odFItN8Hvu2Mas7wuVOT5XZBKPDBfvdaVaqY4k9H2fv01kUlJVvlZPp/HwMCA+j7w/iV4D8t7lURYkmq+jxTShUGR0RW2Y5STywErnJACQHvLGBAR0q7Q3Pckzvi7BB75/Y0YOnv1kNLW5CRiXntFpYHyGg20q1XEnnYGGsMZuP/5U2VZSP/yIFLbt6E2HKV9mgedXEpyJBUOk/lVklHufMRdj/SUNMCcb6gMkuDCJoOGpM8loSuGXFh1v0/2Xy6imUxmnn+cHvnO7R2BWXVV5rOU+9mbAkTkLjgy1ZMk1XI8JNV6WbnI60RfKndyTnTSZVPlbD6O+ucLJaILOV+e4+f7KcsyQEuOvVwu48CBAwDmrmPY/uv3v67gmxRnHjc9oJk+l/XL7wLdQGq1mvqu6emXJPnjPS3ve+l2olsePM/r2M5TtzqwXj6EkZDSr3VmZgYTExOKPHP+
```

Comment:
No you did nothing wrong, the image is gray scale. I applied fake colors to the image after the fact to make it "look good" and pop. So as I see it there are two layers to this problem. 

(1) The color: I can re-upload the example `tif` with a fixed cmap so it opens properly in color, but the `matplotlib` displays don't understand the `cmap`...or at least I don't know how to pass that along to `matplotlib`. In this case, I think it would be best to update the image at the bottom with a gray scale one.

(2) When the final cell runs and the sample data is shown you only see the nuclei. This because the second channel's signal is much lower. Do you think this is also confusing? Should I find another dataset to work with? I'd prefer to not modify the values in the second channel.

Reply:
> the matplotlib displays don't understand the cmap...or at least I don't know how to pass that along to matplotlib.

@elevans See also #24